asistente_ladm_col/gui/supplies/wiz_supplies_etl.py

```diff
@@ -218,6 +219,8 @@ def import_button_clicked(self):
                         res_spatial, msg_spatial = etl.load_spatial_layers()
                         if res_spatial:
                             res_model, msg_model = self.load_model_layers(etl.layers)
+                            self.ladm_layers = [etl.layers[ladm_layer][LAYER] for ladm_layer in etl.layers]
```

Comment:
If there is any problem loading the layers, you won't be able to use them...
Be careful, that's the purpose of the next `if res_model:`.